docs/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Version 3.24.0-eisop2 (August ?, 2022)
 
 **Closed issues:**
 
+eisop#282.
```

Comment:
In `Implementation details`, highlight the changes to the Dataflow Framework.

Reply:
Thanks! I'll add details after finishing the type checking part, currently I am working on iit.